### PR TITLE
Topology tests & allow multicast for log channel

### PR DESCRIPTION
**Pull request by mikeb01**
There is still one outstanding test failure as result of this change.  In `shouldRecoverQuicklyAfterKillingFollowersThenRestartingOne` it appears that when extending the recording the image does not become available to the archive's subscription therefore it never constructs the recording counter and ConsensusModule's conductor times out.

### CODE REVIEW

File: aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModule.java [hunk 1]
```diff
@@ -1306,6 +1308,9 @@ public void conclude()
                 egressPublisher = new EgressPublisher();
             }
 
+            final ChannelUri channelUri = ChannelUri.parse(logChannel());
+            logChannelIsMultiDestination = channelUri.isUdp() && null == channelUri.get(ENDPOINT_PARAM_NAME);
```
Comment (mjpt777) [inline aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModule.java:1312]: Boolean naming convention starts with `is` or `has`, therefore `isLogChannelMultiDestination`.

File: aeron-test-support/src/main/java/io/aeron/test/cluster/TestCluster.java [hunk 2]
```diff
@@ -60,7 +60,9 @@
 {
     private static final int SEGMENT_FILE_LENGTH = 16 * 1024 * 1024;
     private static final long CATALOG_CAPACITY = 128 * 1024;
-    private static final String LOG_CHANNEL = "aeron:udp?term-length=512k";
+//    private static final String LOG_CHANNEL = "aeron:udp?term-length=512k";
+    private static final String LOG_CHANNEL =
+        "aeron:udp?term-length=512k|endpoint=224.20.30.39:24326|interface=localhost";
```
Comment (mjpt777) [inline aeron-test-support/src/main/java/io/aeron/test/cluster/TestCluster.java:65]: Do we always want to test with multicast? Most use will be MDC.

**Comment by mjpt777** [comment]
The general approach looks good.

File: aeron-test-support/src/main/java/io/aeron/test/cluster/TestCluster.java (hunk 2 above)
Comment (mikeb01) [reply aeron-test-support/src/main/java/io/aeron/test/cluster/TestCluster.java]: I wanted to see some runs in CI with multicast enabled.  I've moved it back but left the configuration commented.  I'll pick up a later task to make this a parameter to our cluster tests.

**Pull request merged**